src/AmbientSounds.Uwp/Views/ScreensaverPage.xaml.cs

```diff
@@ -277,6 +276,7 @@ private void RootPage_OnPointerMoved(object sender, PointerRoutedEventArgs e)
         {
             GoBackButton.Visibility = Visibility.Visible;
             ActionButtons.Visibility = Visibility.Visible;
+            CoreWindow.GetForCurrentThread().PointerCursor = new CoreCursor(CoreCursorType.Arrow, 0);
```

Comment:
Found a bug with this:
1. Open screensaver page
2. Let the cursor disappear after a few seconds of no movements
3. Press ESC key. This will navigate Ambie back to the home page
4. Observe the cursor is invisible even in the home page.

Reply:
I’ve set the cursor to re-appear when the page navigates away and also stop the timer